src/stores/my-account.ts

```diff
@@ -147,9 +169,25 @@ export const useMyAccount = create<State & Actions>()((set, get) => ({
         accountStorage.remove()
         accountAddressStorage.remove()
         set({ address: null })
+      } else {
+        // TODO: check how we can do it more elegant
+        const { data: linkedAccounts } =
+          await getLinkedTelegramAccountsQuery.useQuery({
+            address,
+          })
+        userProperties.tgNotifsConnected = (linkedAccounts?.length || 0) > 0
+
+        const [evmLinkedAddress] = await getEvmAddressesFromSubsocial(
```

Comment:
for this one, is there any reason why it must be from blockchain?
maybe you can use the getAccountsData.fetchQuery ?